frontend/src/components/navbar/navstyle.module.scss

```
@@ -0,0 +1,38 @@
+.toolBar{
```

Acordate de mantener los espaciados en las clases:
```suggestion
.toolBar {
```